app/editor/src/features/content/papers/PaperToolbar.tsx

```diff
@@ -41,18 +42,76 @@ export interface IPaperToolbarProps {
 export const PaperToolbar: React.FC<IPaperToolbarProps> = ({ onSearch }) => {
   const [{ filterPaper: filter }, { storeFilterPaper }] = useContent();
   const [{ mediaTypeOptions }] = useLookupOptions();
-  const [{ publishReport }] = useReports();
+  const [{ previewReport, publishReport }] = useReports();
   const [{ publishNotification }] = useNotifications();
   const { morningReportId, frontPageImagesReportId, topStoryAlertId } = useSettings();
   const { toggle, isShowing } = useModal();
 
   const [, setMediaTypeOptions] = React.useState<IOptionItem[]>([]);
   const [sendInfo, setSendInfo] = React.useState<IReportInfo>();
 
+  const [isLoading, setIsLoading] = React.useState(false);
+
   React.useEffect(() => {
     setMediaTypeOptions([new OptionItem<number>('Any', 0), ...mediaTypeOptions]);
   }, [mediaTypeOptions]);
 
+  async function checkReportContent(reportID: number) {
+    try {
+      setIsLoading(true);
+      const preview = await previewReport(reportID);
```

Comment:
Can you check if the report has the message in it "There is no content in this report."?

Reply:
OK. No problem.